src/cogs/janitor/janitor.py

```diff
@@ -7,22 +7,24 @@
 
 class Janitor(commands.Cog):
 
-    def __init__(self, bot: discord.ext.commands.Bot, cache: dict):
+    def __init__(self, bot: discord.ext.commands.Bot):
         self.bot: commands.Bot = bot
-        self.cache: dict = cache
```

Comment:
whyyyyy

Reply:
why not just do self.cache: dict = bot.cache ?

Reply:
so we don't have to bother injecting it everywhere - it's almost the exact same thing.  i also don't know if it's possible to do it this way using load_extension.

Reply:
no i mean we don't need to inject self if we're getting it from bot anyways i agree. just assign it to dict in the constructor. we this all looks good to me